.gitignore

```diff
@@ -73,4 +73,7 @@ es2015/
 !.github/**
```

Comment:
Change [L53](https://github.com/adobe/node-httptransfer/blob/096d1265bb7d7fafe58cfdf281e459eecbf810f0/.gitignore#L53) from `.env` to `*.env`, because I guarantee people will start naming the env files and we want to be sure those are never ever in git.